vendor/bat-native-ledger/src/bat_publishers.cc

```diff
@@ -42,19 +42,18 @@ BatPublishers::~BatPublishers() {
 }
 
 void BatPublishers::calcScoreConsts(const uint64_t& min_duration) {
```

Comment:
related to the comment below, I think we need to specify what these durations are in - `min_duration_seconds` or `min_duration_ms`

Reply:
sure I can rename it

Reply:
done